### PR TITLE
Added Europe Remotely to job boards

### PR DESCRIPTION
**Pull request by IvanJov**
https://github.com/lukasz-madon/awesome-remote-job/issues/312

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,7 +232,9 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Crossover] (https://app.crossover.com/x/marketplace/available-jobs) - Project-based Java, .Net and DevOps positions.
   1. [Datadog](https://www.datadoghq.com/careers/) - Easy to use and scalable monitoring systems for modern and dynamic infrastructure.  Distributed team with offices in New York, Boston and Paris.  Engineers based all around the world.
   1. [DataStax](http://www.datastax.com/company/careers) - Consulting based on Apache Cassandra.
-  1. [Deeson] (https://www.deeson.co.uk/careers) - UK-based w/European team. Digital agency specialising in Drupal, Symfony and Laravel
```
Comment (RichardLitt) [inline README.md]: Can you fix this? Maybe just replace `w/` with `with`, in case other editors mess up here, too.

**Comment by RichardLitt** [comment]
Thanks! I have a couple of questions - I'm not sure this is awesome, at the moment, although it is certainly a remote job board with recent entries, which is good. 

There seem to be only a handful of entries. How often is this updated? Do I get notified of each entry added? Is there a way to look at past entries? Why the low number - is this because these or more quality, or are there not enough resources to get more? 

The link to the job on SO is broken, as well.

**Comment by IvanJov** [comment]
@RichardLitt Thanks for reviewing this! I've updated Deeson in the new commit.

This job board is cool because you can get handpicked remote jobs in your email box and also can check them on a website. As far as I know, it's updated weekly or once in two weeks (as far as I remember). And you are getting a newsletter with a list of latest added jobs.

I have contacted site creator, he will hopefully give us more info here.

**Comment by dabrorius** [comment]
Hey guys, I'm the maintainer of Europe Remotely. Thanks @IvanJov for contacting me. Jobs there are curated manually, only those available to people from European time zones are posted. Newsletter is sent on weekly basis.
The project was a bit neglected in past few months due to my primary job, but I'm going to reserve a significant part of my time for it in the future. Starting from Monday I'm working full-time on it for next several months.

**Comment by RichardLitt** [comment]
Good enough for me. Thank you!

**Comment by dabrorius** [comment]
Thanks!

**Comment by IvanJov** [comment]
@RichardLitt Thanks for merging! And @dabrorius thanks for your explanation! 😊

**Pull request merged**